### ISSUE TITLE
How do set number of indents using CLI?

### ISSUE THREAD

**Issue by rubenaster** [quicktype/quicktype#1651]
Hey folks,

first of all: thanks A LOT for your awesome work! It helps saving so much pain and time!!

So, I've mainly used your web UI to generate Dart classes and now I went over to using CLI with Windows PowerShell. And what I noticed is that the indents went from 2 generated on web to 4 generated with CLI.

![image](https://user-images.githubusercontent.com/3841725/107986724-f3d89100-6fcc-11eb-8488-747da94aef6d.png)

Is there a way to set the indents using the CLI command? I've already checked quicktype --help but didn't find the answer.

Thanks a lot,

Ruben

**Comment by rubenaster** [comment]
Yeah... of course the indents when pasting the code from web are already adjusted by the IDE :)

But still, is there a way to adjust the indents so we can have our auto-generation work and use the generated files directly? Or would we need to adjust the Quicktype code itself for that?

**Comment by dvdsgl** [comment]
We would need to add an option to `quicktype` to do this.
